doc/code/qml_dla.rst

```diff
@@ -0,0 +1,12 @@
+qml.dla
+=======
+
+(Dynamical) Lie Algebra functionality in PennyLane.
+
+See our :doc:`introduction to Dynamical Lie Algebras for quantum practitioners <demos/tutorial_liealgebra>`.
```

Comment:
why is this demo unknown!?
https://pennylane.ai/qml/demos/tutorial_liealgebra/ 🤔

Reply:
is it maybe not part of the docs? Other demos seem to be referenced as "standard" website links, e.g. in `qml_pauli.rst`: 
> For a larger example of how grouping can be used with PennyLane, check out the `Measurement Optimization demo <https://pennylane.ai/qml/demos/tutorial_measurement_optimize/>`_.

Reply:
Those are static links and as far as I know to be avoided. There are plenty of dynamic links to demos in the codebase, e.g. https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/qnn/__init__.py#L21 - hence my confusion 😢

Reply:
Mhm, that _is_ strange. Could this be due to the dynamic links being in `.py` files in the actual `pennylane` directory, whereas `doc/code/qml_dla.rst` is `.rst` outside of that dir? :thinking: Is there a precedence for a dynamic link to a demo in the `.rst` files?

Reply:
That is the only real difference that I am seeing, might be that yeah. I'm gonna try with a static link for now. Maybe the second reviewer has an idea